boards/common/adafruit-nrf52-bootloader/doc.md

```diff
@@ -46,6 +52,12 @@ the bootloader.
 If some other firmware is running or RIOT crashed, you need to enter the bootloader
 manually by double-tapping the board's reset button while the device is connected.
 
+@note Some distributions such as Arch Linux might not have automounting configured.
```

Comment:
I would refrain from linking auto-mounting to the distro. It is just that e.g. Ubuntu has automounting set up be default, and e.g. Arch Linux has not.

It is possible to set up automounting on Arch or to disable it (e.g. for security reasons) on Ubuntu.

Reply:
How about rewording it to "do not setup up automounting in the default installation"?